components/script/dom/eventsource.rs

```diff
@@ -329,10 +329,11 @@ impl FetchResponseListener for EventSourceContext {
     }
 
     fn process_response(&mut self, metadata: Result<FetchMetadata, NetworkError>) {
+        println!("eventsource process response");
```

Comment:
This probably isn't useful? The EventSource API is pretty rare.

Reply:
oops. some of my testing printlns got pushed with my last update...